installation/README.md

```diff
@@ -0,0 +1,93 @@
+# Installing Config Sync manually
+
+This document is a guide on how to manually install Config Sync on a Kubernetes
+cluster without Fleet Management.
+
+The recommended approach is to [install Config Sync managed by fleet](https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/how-to/installing-config-sync).
+
+This document includes configurations for some common user journeys, but is not
+an exhaustive list of how a Config Sync installation can be customized.
+
+## Pre-requisites
+
+This guide requires the following command line tools:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [kustomize](https://github.com/kubernetes-sigs/kustomize)
+
+## Uninstall Config Management
+
+**Prerequisites**:
+- Set the kubectl context in your shell to the desired cluster before proceeding.
+- Ensure [hierarchy controller is disabled](https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/how-to/migrate-hierarchy-controller#kubectl) before proceeding.
+
+
+If you have previously installed Config Management on your cluster, Config Management
```

Comment:
Use `ConfigManagement` resource to make it more clear you're talking about a CRD

Reply:
My intent was to refer to the product more broadly rather than just the CRD. Config Management consists of the various subcomponents (operator deployment, SA, CRB, CRD, etc), where the CRD is just one of the subcomponents. Would you prefer the language be more specifically referring to the CRD?

Reply:
I'm not sure if customers understand what "install Config Management" means. Some might think it == installing Config Sync